plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -125,16 +122,27 @@ function compile(err, stats) {
   }
 };
 
-if (args['watch']) {
-  config.watch = true;
-  config.mode = 'development';
-  process.env['BABEL_ENV'] = 'development';
-  copyToPluginsFolder(config);
-  webpack(config, compile);
-} else if (args['_'][0] === 'init') {
-  process.exitCode = init(args['_'][1]);
-} else {
-  process.env['BABEL_ENV'] = 'production';
-  copyToPluginsFolder(config);
-  webpack(config, compile);
-}
+
+const argv = yargs(process.argv.slice(2))
+  .command(['build', '$0'], 'Build the plugin', {}, (argv) => {
+    process.env['BABEL_ENV'] = 'production';
+    copyToPluginsFolder(config);
+    webpack(config, compile);
+  })
+  .command('watch', 'Watch for changes and build the plugin', {}, (argv) => {
+    config.watch = true;
+    config.mode = 'development';
+    process.env['BABEL_ENV'] = 'development';
+    copyToPluginsFolder(config);
+    webpack(config, compile);
+  })
+  .command('init <name>', 'Initialize a new plugin folder with base code', (yargs) => {
```

Comment:
I think from a product point of view, doing `headlamp-plugin create NAME` sounds better. Init is more ambiguous.

Reply:
This is changed to create.